packages/block-library/src/blocks/supernova/scss/_card.scss

```diff
@@ -106,7 +106,11 @@
   position: relative;
   background-color: var(--novablocks-bg-color);
   padding: var(--novablocks-card-image-padding, 0);
-  opacity: var(--collection-card-media-opacity, 1);
+  //opacity: var(--collection-card-media-opacity, 1);
+  
+  // @todo: Remove old CSS related
```

Comment:
well?

Reply:
I didn't removed that, because there is still `Card Media Opacity` and I was waiting to see exactly if you removed it on your working branch.